src/main/resources/org/apache/camel/language/datasonnet/datasonnet.json

```diff
@@ -7,13 +7,13 @@
     "description": "To use Datasonnet scripts in Camel expressions or predicates.",
     "deprecated": false,
     "deprecationNote": "language,datasonnet",
-    "firstVersion": "1.0.0",
+    "firstVersion": "3.3.0",
     "label": "language,datasonnet",
     "javaType": "com.modus.camel.datasonnet.language.DatasonnetLanguage",
     "modelJavaType": "com.modus.camel.datasonnet.language.model.DatasonnetExpression",
```

Comment:
I think this needs updated to remove `model` from the packages to reflect the move that's also in this PR

Reply:
Not sure if the entire "modelJavaType" property is still needed. Can you please clarify?

Reply:
I suppose it's not needed, but I don't know to be honest. I'm not entirely sure on the purpose of this file in the camel component... paradigm(?)

Reply:
ok, I think I know what's going on. Model is the class used for the XML schemas in Camel - for example, here's the one for Groovy (note it belongs to the camel-core, not the camel-groovy module) - https://github.com/apache/camel/blob/master/core/camel-core-engine/src/main/java/org/apache/camel/model/language/GroovyExpression.java

And the entry in JSON file points to it: https://github.com/apache/camel/blob/831458ec33a76e5e1f8ad748658581c0dde02e7d/components/camel-groovy/src/generated/resources/org/apache/camel/language/groovy/groovy.json#L16

I'd say let's keep this line as is and let's not delete the model class - just in case.